test/classes.coffee

```diff
@@ -739,21 +742,21 @@ test "#2599: other typed constructors should be inherited", ->
   ok (new Derived) not instanceof Base
   ok (new Base) not instanceof Base
 
-test "#2359: extending native objects that use other typed constructors requires defining a constructor", ->
-  class BrokenArray extends Array
-    method: -> 'no one will call me'
-
-  brokenArray = new BrokenArray
-  ok brokenArray not instanceof BrokenArray
-  ok typeof brokenArray.method is 'undefined'
-
-  class WorkingArray extends Array
-    constructor: -> super
-    method: -> 'yes!'
-
-  workingArray = new WorkingArray
-  ok workingArray instanceof WorkingArray
-  eq 'yes!', workingArray.method()
+# test "#2359: extending native objects that use other typed constructors requires defining a constructor", ->
```

Comment:
This is broken now, since `brokenArray instanceof BrokenArray` and `brokenArray instanceof Array` with ES2015 inheritance :D